tracer/build/PackageVersionsGeneratorDefinitions.json

```diff
@@ -436,7 +436,7 @@
     "SampleProjectName": "Samples.CosmosDb",
     "NugetPackageSearchName": "Microsoft.Azure.Cosmos",
     "MinVersion": "3.6.0",
-    "MaxVersionExclusive": "4.0.0",
+    "MaxVersionExclusive": "3.42.1",
```

Comment:
I tested version `3.42.0` locally.
The snapshots aren't entirely out of date as some `git` tags have been scrubbed from all snapshots but besides that it looked good, will make another PR to update those snapshots though.

Reply:
🤔 Is this how we _want_ it to work 🤔 I don't know if the updater would notify about new versions in that case? I'll take your word for it that this is how you _want_ it to work though 🙂 

Another possibility - we add a flag to the pipeline to make it test cosmos on _these_ PRs (version bump) only. We only don't test it in CI because the infra is really flaky IIRC, but that doesn't matter so much if it only affects these PRs 🤷‍♂️

Reply:
So it is how the `Azure.Messaging.ServiceBus` is currently setup as well, my thoughts is that hopefully when Dependabot starts back up it should be able to keep on top of things as that should update it as well.

I do like the idea of adding that flag though, will look into that thanks!